src/nvim/os/env.c

```diff
@@ -2,10 +2,20 @@
 
 #include <uv.h>
 
+// vim.h must be included before charset.h (and possibly others) or things
+// blow up
```

Comment:
Is this actually true? I would imagine that generated declarations would take care of this, although I'm not sure.